### PR TITLE
non-member license commitment

### PR DESCRIPTION
**Pull request by wseltzer**
Proposal to address https://github.com/w3c/w3process/issues/67

### CODE REVIEW

**Comment by dwsinger** [comment]
I do not understand "other than an error correction". We want licenses for technical changes, including fixes. Classes 1 and 2 are the editorial changes. What am I missing?

File: index.html [hunk 1]
```diff
@@ -2105,7 +2105,13 @@ <h4 id="correction-classes">6.2.5 Classes of Changes</h4>
         <dt>4. New features</dt>
         <dd>Changes that add a new functionality, element, etc.</dd>
       </dl>
-
+	    <h4 id="contributor-license">6.2.X Contributor License Grants</h4>
+	    <p>When a contributor who is not a Working Group participant offers a change in class 3 or 4 (as described in <a href="correction-classes"">6.2.5</a>)
```
Comment (frivoal) [inline index.html]: syntax error: there's one double-quote to many after correction-classes

**Comment by nrooney** [comment]
Agreed not to add in meeting on 15/08/18

**Comment by wseltzer** [comment]
I thought we agreed to make changes, which I just made.

**Comment by wseltzer** [comment]
or, see https://github.com/w3c/w3process/pull/200

**Comment by frivoal** [comment]
#200 was not set up as an alternative to this pull request, but as a complement to this one. Merging it (which is now done) did not add the whole fix to the main branch, but the complementary fix to this branch.

So we now need to merge this. I believe this is merely a technicality, and that we did approve of it (see discussion in #200 and cg meeting minutes), so the editor should go ahead and merge this.

**Pull request merged**